.circleci/config.yml

```diff
@@ -123,6 +123,9 @@ workflows:
             parameters:
               jdk: ["8", "11", "17"]
               kotlin: ["1.5.32", "1.6.21", "1.7.20", "1.8.0"]
+            exclude:
```

Comment:
Some weird interplay here I didn't want to deal with.

Reply:
we should probably drop support for kotlin 1.5 at this point

Reply:
dropped